test/katello_test_helper.rb

```diff
@@ -69,7 +69,6 @@ module FixtureTestCase
   included do
     extend ActiveRecord::TestFixtures
 
-    self.use_transactional_fixtures = true
```

Comment:
should this be changed to use_transactional_tests ?

Reply:
In Foreman this got only removed, as it's not changing anything with Rails 5.x.

What we have in core:
```
$ git grep use_transaction
test/integration_test_helper.rb:  self.use_transactional_tests = false
test/unit/tasks/seeds_test.rb:  self.use_transactional_tests = false
```

Reply:
OK, found https://blog.bigbinary.com/2016/05/26/rails-5-renamed-transactional-fixtures-to-transactional-tests.html "By default this functionality is enabled. "